Add-ons/UmbracoForms/Developer/Prepping-Frontend/index-v9.md.md

```diff
@@ -0,0 +1,66 @@
+---
+versionFrom: 9.0.0
+---
+
+# Preparing your Frontend
+
+For Umbraco Forms to work correctly, you need to include some client dependencies.
+
+## Umbraco Forms version 9.0.0
+
+You can use the following Razor helper to output script tags containing the dependencies. To access this method you will need a reference to `Umbraco.Forms.Mvc`:
+
+```html
+@using Umbraco.Forms.Web
+<head>
+    @Html.RenderUmbracoFormDependencies()
+</head>
+```
+
+Alternatively, you can add the dependencies to the body tag:
+
+```html
+@using Umbraco.Forms.Web
+...
+
+<body>
+    @Html.RenderUmbracoFormDependencies()
+</body>
+```
+
+All dependencies originate from your Umbraco Forms installation, which means that no external references are needed.
+
+The easiest way to add the dependencies is to fetch them from a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network). There are various CDN services you can use:
```

Comment:
```suggestion
## Validation Using jQuery

If you want to use jQuery as your validation framework for Umbraco Forms, you can manually add the following client dependencies without using the above Razor method:

- `jQuery` (JavaScript library)
- `jQuery validate` (jQuery plugin that provides client-side Form validation)
- `jQuery validate unobtrusive` (Add-on to jQuery Validation that provides unobtrusive validation via data-* attributes)

The easiest way to add the dependencies is to fetch them from a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network). There are various CDN services you can use:
```

Think we need to add this sub-heading and restore the paragraph, so it's clear why we are now talking about jQuery.